server/runtime/src/main/java/org/infinispan/server/Server.java

```diff
@@ -470,7 +470,11 @@ public void setExitHandler(ExitHandler exitHandler) {
          log.serverFailedToStart(Version.getBrandName(), e);
```

Comment:
you probably want to remove this line. no need to log the exception twice.

Reply:
Indeed, fixed! I went back-and-forth about logging the error in Server or in Bootstrap, and I ended up missing the extra logging here.